src/main/java/org/apereo/openlrs/storage/noop/NoOpTierTwoStorage.java

```diff
@@ -45,6 +45,12 @@ public OpenLRSEntity findById(String id) {
 	}
 
 	@Override
+	public OpenLRSEntity findBySourceId(String sourceId) {
+		log.warn("Using NoOp Tier Two Storage");
```

Comment:
I'm just following the example set by other OpenLRS developers.

I didn't want to change this class at all because I've not been using noop storage.  However, I changed interface `TierTwoStorage` for use in class `NormalizedMongoTierTwoStorage`.  Since this class, `NormalizedElasticsearchTierTwoStorage`, and `XApiOnlyElasticsearchTierTwoStorage` all implement `TierTwoStorage`, I was forced to make them agree with the interface or else the application wouldn't build.